### ISSUE TITLE
[PROD] query show error

### ISSUE THREAD

**Issue by ppazos** [ppazos/cabolabs-ehrserver#839]
```

URI
    /query/show
Class
    java.lang.IllegalArgumentException
Message
    Error processing GroovyPageView: Null value was assigned to a property of primitive type setter of com.cabolabs.ehrserver.query.datatypes.DataCriteriaDV_QUANTITY.magnitudeNegation
Caused by
    null

Line | Method
->>  111 | doFilter  in /query/show.gsp
- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - 

Caused by PropertyAccessException: Null value was assigned to a property of primitive type setter of com.cabolabs.ehrserver.query.datatypes.DataCriteriaDV_QUANTITY.magnitudeNegation
->>  101 | doCall    in gsp_ehr_queryshow_gsp$_run_closure2
- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - 
|    175 | run       in gsp_ehr_queryshow_gsp
|    198 | doFilter  in grails.plugin.cache.web.filter.PageFragmentCachingFilter
|     63 | doFilter  in grails.plugin.cache.web.filter.AbstractFilter
|     53 | doFilter  in grails.plugin.springsecurity.web.filter.GrailsAnonymousAuthenticationFilter
|     62 | doFilter  in grails.plugin.springsecurity.web.authentication.logout.MutableLogoutFilter
|   1149 | runWorker in java.util.concurrent.ThreadPoolExecutor
|    624 | run       in java.util.concurrent.ThreadPoolExecutor$Worker
^    748 | run . . . in java.lang.Thread

Caused by IllegalArgumentException: null
->>  101 | doCall    in gsp_ehr_queryshow_gsp$_run_closure2
- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - 
|    175 | run       in gsp_ehr_queryshow_gsp
|    198 | doFilter  in grails.plugin.cache.web.filter.PageFragmentCachingFilter
|     63 | doFilter  in grails.plugin.cache.web.filter.AbstractFilter
|     53 | doFilter  in grails.plugin.springsecurity.web.filter.GrailsAnonymousAuthenticationFilter
|     62 | doFilter  in grails.plugin.springsecurity.web.authentication.logout.MutableLogoutFilter
|   1149 | runWorker in java.util.concurrent.ThreadPoolExecutor
|    624 | run       in java.util.concurrent.ThreadPoolExecutor$Worker
^    748 | run . . . in java.lang.Thread

```

**Comment by ppazos** [comment]
This error occurs because of the OPT manager, it seems the OPT is not loaded and it fails con DataCriteriaDV_QUANTITY.criteriaSpec on this line:

`optMan.getNode(archetypeId, path, namespace)?.xmlNode.list.each {`

That is why it says "can't call list on null".

This might be linked to https://github.com/ppazos/cabolabs-ehrserver/issues/838

**Comment by ppazos** [comment]
This is fixed because #838 was fixed and was the cause of this error.
